### DISCUSSION TITLE
Dữ liệu các mã hủy niêm yết, chuyển sàn

### DISCUSSION THREAD

**Discussion by TungPhaSanh** [thinh-vu/vnstock#130] [Đóng góp ý tưởng (Improvement ideas)]
Hi thinh-vu,

Cảm ơn bạn với sản phẩm thực sự hữu ích. Mình muốn có một chút đóng góp tới sản phẩm, hi vọng bạn có thể upgrade trong các version tiếp theo.
Hiện tại với nhu cầu của mình để làm backtest quá khứ, mình mong muốn có dữ liệu về sàn giao dịch của các mã ở một ngày cụ thể, để biết rằng cổ phiếu đó có chuyển sàn hay không. Ngoài ra, mình cũng mong muốn có đủ dữ liệu các mã đã bị hủy niêm yết, để làm backtest một cách khách quan và chính xác nhất. 

Hi vọng được bạn đón nhận và cải tiến sản phẩm, để chúng ta có thể có một database dữ liệu đầy đủ và chính xác hơn!

**Comment by thinh-vu** [comment]
Chào @TungPhaSanh 
Cám ơn bạn đã đóng góp ý kiến xây dựng sản phẩm. Dữ liệu này sẽ được bổ sung vào thư viện vnstock trong thời gian tới. Bạn cũng có thể tham gia gói thành viên để được hỗ trợ nhanh chóng. 
Chúc bạn sử dụng vnstock thành công!
